src/WriteupServiceProvider.php

```diff
@@ -19,7 +19,7 @@ public function boot(Router $router)
         }
 
 
-        if (config('writeup.run_in_production')) {
+        if (config('app.env' === 'production') && config('writeup.run_in_production')) {
```

Comment:
This will ignore the development mode now.
-> What if someone set run_in_production to true but the app in development mode.
The code wouldn't run.